test.js

```diff
@@ -292,6 +293,74 @@ test('no auto logging with autoLogging set to false', function (t) {
   }, handle)
 })
 
+test('no auto logging with autoLogging set to true and path ignored', function (t) {
+  var dest = split(JSON.parse)
+  var logger = pinoHttp({
+    autoLogging: true,
+    autoLoggingOptions: {
+      ignorePaths: ['^/ignorethis$']
```

Comment:
Should there be a test that includes a regex literal?

Reply:
this gets solved with next review (where regex are no longer used)